### DISCUSSION TITLE
Support to sign commit by gpg-private-key

### DISCUSSION THREAD

**Discussion by qhy040404** [lowlighter/metrics#1330] [Ideas]
We can use GPG Private key to sign commits. Not necessary, but maybe useful?

**Comment by lowlighter** [comment]
Hi 👋 !

Not sure what you meant 😅  

If you're talking about displaying how many commits has been done with verified GPG than it's supposed to be already supported by language plugin here:
https://github.com/lowlighter/metrics/blob/520ff377142700a2e98fb3ee39798b6b8c43fe98/source/templates/classic/partials/languages.ejs#L82-L89
It was something similiar to 
![image](https://user-images.githubusercontent.com/22963968/209456107-745706e7-2998-442b-982a-f231867e8356.png)


But it's true I haven't seen this from some time, and it's not on the examples anymore. 
Maybe there was a regression 🤔 ?

Else if you're talking about something else, can you elaborate on what you'd like to see ?

**Comment by qhy040404** [comment]
I mean we can sign the generated `metrics.svg` before it is commited. 
![image](https://user-images.githubusercontent.com/45379733/209456287-b6f023bb-2a0d-47f7-ad05-2cc7adc88d1c.png)

**Reply by lowlighter** [reply]
So you'd like to be able to provide the GPG private key of your bot [@qhybot](https://github.com/qhyBot) so it can make verified commits ?

**Reply by qhy040404** [reply]
Yes 🥺
